File: pgm/ABC_Kotlin/ABC_392_A.kt
// import java.time.*

// import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"
const val space: String = " "
const val one: Int = 1
const val two: Int = 2
const val zero: Int = 0

fun main() {
    val A: List<Int> = readln().split(space).map { it.toInt() }.sorted()
    if (A[zero] * A[one] == A[two]) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}

File: pgm/ABC_Dart/ABC_392_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';

void main() {
  final List<int> A = stdin.readLineSync()!.split(" ").map(int.parse).toList();
  A.sort();
  const String Yes = 'Yes';
  const String No = 'No';
  const int one = 1;
  const int two = 2;
  const int zero = 0;
  if (A[zero] * A[one] == A[two]) {
    print(Yes);
  } else {
    print(No);
  }
  exit(0);
}
